build3/premake4.lua

```diff
@@ -49,43 +49,55 @@
 
 --	--_OPTIONS["midi"] = "1";
 
-	newoption
-	{
-		trigger = "bullet2demos",
-		description = "Compile the Bullet 2 demos (Demo/Extra folder)"
-	}
+    newoption
+    {
+        trigger = "no-demos",
+        description = "Don't build demos"
+    }
 
 	newoption
 	{
 		trigger = "enet",
 		description = "Enable enet NAT punchthrough test"
 	}
+
 	newoption
 	{
 		trigger = "lua",
 		description = "Enable Lua scipting support in Example Browser"
 	}
 
-	 newoption {
-    trigger     = "targetdir",
-    value       = "path such as ../bin",
-    description = "Set the output location for the generated project files"
-  }
-	
-	
+	newoption {
+        trigger     = "targetdir",
+        value       = "path such as ../bin",
+        description = "Set the output location for the generated project files"
+    }
+
 	newoption
 	{
-		trigger = "without-gtest",
+		trigger = "no-test",
+		description = "Disable all tests"
+	}
+
+	newoption
+	{
+		trigger = "no-gtest",
 		description = "Disable unit tests using gtest"
 	}
```

Comment:
Note: the **--without-gtest** option was replaced with **--no-gtest**.  If this causes a problem we can roll this change back.